Release Notes/511.md

```diff
@@ -35,6 +35,10 @@
 - `String.isValidIdentifier(for:)`
   - Description: `SwiftParser` adds an extension on `String` to check if it can be used as an identifier in a given context.
   - Pull Request: https://github.com/apple/swift-syntax/pull/2434
+  
+- `FixItApplier`
+  - `FixItApplier` can apply edits or Fix-Its to a syntax tree and return the fixed-up source
+  - Pull Request: xxx
```

Comment:
I'm leaving this suggestion here in case you overlook it before merging:
```suggestion
  - Pull Request: https://github.com/apple/swift-syntax/pull/2467
```